test/architecture-tests/src/test/java/software/amazon/awssdk/archtests/CodingConventionWithSuppressionTest.java

```diff
@@ -48,7 +48,8 @@ public class CodingConventionWithSuppressionTest {
     private static final Set<Pattern> ALLOWED_WARN_LOG_SUPPRESSION = new HashSet<>(
         Arrays.asList(ArchUtils.classNameToPattern(EmfMetricLoggingPublisher.class),
                       ArchUtils.classNameToPattern(MetricEmfConverter.class),
-                      ArchUtils.classNameToPattern(MakeHttpRequestStage.class)));
+                      ArchUtils.classNameToPattern(MakeHttpRequestStage.class),
+                      ArchUtils.classNameToPattern("software.amazon.awssdk.services.s3.internal.crt.S3CrtResponseHandlerAdapter")));
```

Comment:
Passing `S3CrtResponseHandlerAdapter.class` does not work because it'd try to load CRT classes and fail and we don't want to include CRT dependency in our arch tests